package.json

```diff
@@ -20,19 +20,15 @@
     "test": "mocha",
     "posttest": "npm run lint & npx tsc --noEmit --project tsconfig.dev.json",
     "test6": "npm run build && mocha",
-    "build-es2015": "tsc --target es2015 --outDir es2015",
```

Comment:
Could we also have es2017 so that the sdk & UI can also update w/o needing an extra major?

Reply:
Can they use babel or similar to downlevel it if they need to? I wouldn't be surprised if that's already necessary for quite a lot of dependencies at this point

Reply:
Also the SDK is probably going to need a major anyway to bump pinejs-client isn't it? Since it exposes it as `sdk.pine` (although maybe that's not part of the supported contract)